osquery/filesystem/windows/fileops.cpp

```diff
@@ -98,6 +98,43 @@ class WindowsFindFiles {
   fs::path path_;
 };
 
+Status windowsShortPathToLongPath(const std::string& shortPath,
+                                  std::string& rLongPath) {
+  TCHAR longPath[MAX_PATH];
```

Comment:
You could use `std::array<TCHAR>(MAX_PATH);` then `std::move` it into the string. Or do the old `&string[0]` trick to access a non-const buffer.